react/src/views/BoxEdit/components/BoxEdit.tsx

```diff
@@ -263,6 +277,36 @@ function BoxEdit({
               )}
             />
           </ListItem>
+          <ListItem>
+            <Controller
+              control={control}
+              name="tags"
+              render={({
+                field: { onChange, onBlur, name, value, ref },
+                fieldState: { error },
+              }) => (
+                <FormControl isInvalid={!!error} id="tags">
+                  <FormLabel>Tags</FormLabel>
+                  <Box border="2px">
```

Comment:
Is it true that you don't use the tag color here?

Reply:
@pylipp Yes, I want to use tag color and display tags with background colors in the tags selection input, but that turns out to be more complex and outside the scope of this PR, but I keep it on query so we can implement it later.

Reply:
This looks perfect and easy for adding color to tags:
https://github.com/anubra266/choc-autocomplete#multi-select-with-tags

Reply:
@HaGuesto Sure, I'll check this out, but I think it could also be done as a separate PR